### DISCUSSION TITLE
App Doesn't Recognize When Files Are Deleted Using File Manager

### DISCUSSION THREAD

**Discussion by Budlyte** [advplyr/audiobookshelf-app#1238] [Q&A]
If I delete local files using a file manager then the ABS app doesn't pick up that those files are now gone and can't play the item. Attempting to delete that item, which is already gone, results in an error about deleting the file.  Essentially I'm stuck unable to even stream a book that has been deleted from my phone by another app.

**Comment by nichwall** [comment]
Open feature request: https://github.com/advplyr/audiobookshelf-app/issues/1227
